requirements.txt

```diff
@@ -40,6 +40,7 @@ neocore==0.4.10
 numpy==1.14.5
 pbr==4.0.4
 peewee==2.10.2
+pexpect==4.2.1
```

Comment:
put this into `requirements_dev.txt` please. the `requirements.txt` file should only contain requirements for running it as user :)

Reply:
Okay thank you for clarifying.

Reply:
Do you mean requirements_doc.txt or should I create a new file? I do not see a requirements_dev.txt

Reply:
just to note -- after internal talks we leave it like this ;)